sdk/loadtestservice/Azure.Developer.LoadTesting/src/Models/TestV2.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+using System.IO;
+using System.Text.Json;
+using System;
+using Azure.Core.Json;
+using Azure.Core;
+
+namespace Azure.Developer.LoadTesting.Models
+{
+    /// <summary>
+    ///  model TestV2 {
+    ///    @key
+    ///    key: string; // Because it has @key so there will be a @path implictly added to it.
+    ///
+    ///    @visibility("read")
+    ///    readOnlyProperty: string; // Add this propery to differentiate request from response.
+    ///    optionalProperty?: string;
+    ///    requiredProperty: int32;
```

Comment:
We should understand why this property is required for GET and PUT but not for PATCH.  I don't think we have a good foundation for this question yet.